### PR TITLE
Test placeholder is removed when editor is changed and it has content

### PR DESCRIPTION
**Pull request by antoniopol06**

### CODE REVIEW

File: src/plugins/placeholder.js [hunk 1]
```diff
@@ -32,6 +32,7 @@
              */
             init: function(editor) {
                 editor.on('blur', this._checkEmptyData, this);
+                editor.on('change', this._checkEmptyData, this);
```
Comment (jbalsas) [inline src/plugins/placeholder.js]: We probably want to debounce this, don't we?

**Comment by ipeychev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/plugins/placeholder.js (hunk 1 above)
Comment (ipeychev) [inline src/plugins/placeholder.js]: However, after debouncing the test will fail.

**Comment by ipeychev** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/liferay/alloy-editor/compare/25be6c3~3...25be6c3) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**